src/main/kotlin/org/jitsi/nlj/util/Bandwidth.kt

```diff
@@ -24,11 +24,9 @@ import kotlin.math.sign
  * [Bandwidth] models a current bandwidth, represented as a rate
  * of bits per second.
  */
-class Bandwidth(bps: Double) : Comparable<Bandwidth> {
-    val bps: Double = bps
-
-    val kbps: Double = bps / 1000
-    val mbps: Double = bps / (1000 * 1000)
+inline class Bandwidth(val bps: Double) : Comparable<Bandwidth> {
+    fun kbps(): Double = bps / 1000
+    fun mbps(): Double = bps / (1000 * 1000)
```

Comment:
I think I like these as properties with getters rather than methods - otherwise you have `bps` but `kbps()` and `mbps()`, which is annoyingly asymmetric.

Reply:
Yeah I fixed these in a later commit.

Reply:
Need to update `BandwidthTest` accordingly.

Reply:
Oops, fixed.